src/main/java/com/tournament/cup/play/Tournament.java

```diff
@@ -0,0 +1,58 @@
+package com.tournament.cup.play;
+
+import com.tournament.cup.details.Squad;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@AllArgsConstructor
+public class Tournament {
+
+    private TeamsListFour teamsListFour;
+
+    public String getWinnerFour(Squad squadFirstF, Squad squadSecondF, String four) {
+
+        String resultFour = four + ", " + squadFirstF.getTeamName() + ",  " + squadSecondF.getTeamName() + ", " + squadFirstF.getPoints() + " : " + squadSecondF.getPoints();
+        if (squadFirstF.getPoints() == squadSecondF.getPoints()) {
+            teamsListFour.deleteTeamFour(squadFirstF);
+        } else if (squadFirstF.getPoints() > squadSecondF.getPoints()) {
+            teamsListFour.deleteTeamFour(squadSecondF);
+        } else if (squadFirstF.getPoints() < squadSecondF.getPoints()) {
+            teamsListFour.deleteTeamFour(squadFirstF);
+        }
+        return resultFour;
+    }
+
+    private TeamsListEight teamsListEight;
+
+    public String getWinnerEight(Squad squadFirstE, Squad squadSecondE, String eight) {
```

Comment:
This method looks almost identical to getWinnerFour(). Consider using loop instead